quil-rs/src/instruction/gate.rs

```diff
@@ -72,14 +72,17 @@ pub enum GateError {
         parameters: Vec<Expression>,
     },
 
-    #[error("cannot produce a matrix for gate `{name}` with variable qubit {qubit}")]
+    #[error("cannot produce a matrix for gate `{name}` with variable qubit {qubit}", qubit=.qubit.to_quil_or_debug())]
```

Comment:
`qubit=.qubit.to_quil_or_debug()`

Is that extra period a mistake, or special thiserror syntax?

Reply:
It's correct, that's `thiserror`